src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+const Calculator = () => (
```

Comment:
- [ ] This component should have at least one child component which you would import here in this file and use it in your Calculator component as it is the project requirement.
  - Create at least 1 child component

Reply:
Okay 👍